Client/Frontend/Browser/Tabs/TabTrayViewController.swift

```diff
@@ -15,13 +15,23 @@ protocol TabTrayController: UIViewController,
 }
 
 class TabTrayViewController: UIViewController, TabTrayController {
+    struct UX {
+        struct NavigationMenu {
+            static let height: CGFloat = 32
+            static let width: CGFloat = 343
```

Comment:
`width` is not used yet?

Reply:
Not yet, I just copy the whole struct but I can remove until is actually needed